src/core/components/courses/ProfileTraineeFollowUp.vue

```diff
@@ -161,23 +161,11 @@ export default {
           ...(this.isClientInterface && { company: this.loggedUser.company._id }),
         };
         const unsubscribedAttendancesGroupedByTrainees = await Attendances.listUnsubscribed(query);
-        const formattedUnsubscribedAttendances = Object.keys(unsubscribedAttendancesGroupedByTrainees)
+        this.unsubscribedAttendances = Object.keys(unsubscribedAttendancesGroupedByTrainees)
           .map(traineeId => ({
-            _id: unsubscribedAttendancesGroupedByTrainees[traineeId][0].trainee._id,
-            trainee: unsubscribedAttendancesGroupedByTrainees[traineeId][0].trainee,
-            attendances: unsubscribedAttendancesGroupedByTrainees[traineeId].map(a => ({
-              _id: a._id,
-              duration: this.getSlotDuration(a.courseSlot),
-              slot: { startDate: a.courseSlot.startDate, endDate: a.courseSlot.endDate },
-              trainer: formatIdentity(get(a, 'trainer.identity'), 'FL'),
-              misc: a.misc,
-            })),
+            _id: traineeId,
+            ...this.formatTraineeAttendances(unsubscribedAttendancesGroupedByTrainees[traineeId]),
```

Comment:
quitte a faire ca, tu pourrais carrement passer 2 arguments `unsubscribedAttendancesGroupedByTrainees` et `traineeId` pour mettre le `_id: traineeId,` dans la methode de formatage
et comme ca tu as `.map(traineeId => this.formatTRaineeAttendances(unsubscribedAttendancesGroupedByTrainees, traineeId)`

Reply:
done